MTFO/Custom/Scans.cs

```diff
@@ -31,11 +31,14 @@ public struct BioScanGx
 
         public class BioScanColorByMode
         {
+            // fix dak's skill issue :)
+#pragma warning disable IDE1006 // Naming Styles
```

Comment:
If it's caps issue I think you could just change it to PascalCase since `PropertyNameCaseInsensitive` setting for json serializer is true for default